libretro/vkbd.i

```diff
@@ -158,6 +158,10 @@ const char* keyDesc[] = {
 "RETROK_x",//120,
 "RETROK_y",//121,
 "RETROK_z",//122,
+"RETROK_LEFTBRACE",//123,
```

Comment:
Adding keys here will remove keys from the bottom of the selection. Only 127 total keys are allowed in the mapper_* options. Is this really what we want to have these keys instead of META etc?

Reply:
Ok, let's not add them then.